### PR TITLE
check lego: AWS List Unused Secrets

### PR DESCRIPTION
**Pull request by harshal-netserv**
## Description
AWS List Unused Secrets

### Testing
image

### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

**Review by dougsillars — APPROVED** [review]

File: AWS/legos/aws_list_unused_secrets/aws_list_unused_secrets.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                        secret_dict["secret_name"] = secret_id
+                        secret_dict["region"] = reg
+                        result.append(secret_dict)
+        except Exception as e:
+            pass
+
+    if len(result) != 0:
+        return (False, result)
+    else:
+        return (True, [])
```
Comment (shloka-bhalgat-unskript) [inline AWS/legos/aws_list_unused_secrets/aws_list_unused_secrets.py:72]: Should return None

**Review by dougsillars — DISMISSED** [review]

**Comment by shloka-bhalgat-unskript** [comment]
@harshal-netserv legoschema is failing, can you take a look at this?

**Comment by shloka-bhalgat-unskript** [comment]
@harshal-netserv run legoschema is still failing , can you take a look?

**Review by amit-chandak-unskript — APPROVED** [review]

File: AWS/legos/aws_list_unused_secrets/aws_list_unused_secrets.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "action_title": "AWS List Unused Secrets",
+    "action_description": "This action lists all the unused secrets from AWS by comparing the last used date with the given threshold.",
+    "action_type": "LEGO_TYPE_AWS",
+    "action_entry_function": "aws_list_unused_secrets",
+    "action_needs_credential": true,
+    "action_output_type": "ACTION_OUTPUT_TYPE_OBJECT",
+    "action_is_check":true,
+    "action_supports_poll": true,
+    "action_supports_iteration": true,
+    "action_categories": [ "CATEGORY_TYPE_IAM", "CATEGORY_TYPE_SECOPS", "CATEGORY_TYPE_COST_OPT"],
```
Comment (dougsillars) [inline AWS/legos/aws_list_unused_secrets/aws_list_unused_secrets.json:11]: can you add a category CATEGORY_TYPE_AWS?

**Pull request merged**